boa/src/builtins/object/mod.rs

```diff
@@ -406,6 +407,20 @@ pub fn make_object(_: &mut Value, args: &[Value], ctx: &mut Interpreter) -> Resu
     Ok(object)
 }
 
+/// Uses the SameValue algorithm to check equality of objects
+pub fn is(_: &mut Value, args: &[Value], _: &mut Interpreter) -> ResultValue {
+    let x = args.get(0).cloned().unwrap_or_else(Value::undefined);
+    let y = args.get(1).cloned().unwrap_or_else(Value::undefined);
```

Comment:
Maybe we should have a test case for when the arguments are not specified, like the previous comment.